scripts/listen.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+require(__dirname + '/../' + 'constants');
+
+const requested = process.env.npm_config_collection;
```

Comment:
Also, 'requested' seems ambiguous for a variable name. It could be renamed to something more meaningful.